src/Buildalyzer/AnalyzerResult.cs

```diff
@@ -89,6 +89,13 @@ internal AnalyzerResult(string projectFilePath, AnalyzerManager manager, Project
                 .Select(x => x.Item2)
                 .ToArray() ?? Array.Empty<string>();
 
+        public string[] PreprocessorSymbols =>
+            _cscCommandLineArguments
+                ?.Where(x => x.Item1 == "define")

```

Comment:
Will it make sense to compare things ignoring the case?

Reply:
As far as I can see all Item1 values are lowercase. They are gathered from msbuild which doesn't really change I think. But I guess it couldn't hurt to add ignoring case. But then we should do this for all cscCommandLineArguments we use in this file